### PR TITLE
Fix reference to emscripten-site in about page

### PR DESCRIPTION
**Pull request by danieloskarsson**

### CODE REVIEW

**Comment by danieloskarsson** [comment]
Does this small change require that I add myself to AUTHORS as described in https://emscripten.org/docs/contributing/developers_guide.html?
It seems I don't deserve to be in AUTHORS for such a small contribution, but if that's required for legal reasons just let me know and I will update this PR.

File: site/source/docs/site/about.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ Building the site
 
 The site sources are stored on `GitHub <https://github.com/emscripten-core/emscripten/tree/incoming/site>`_. Edits and additions should be submitted to this branch in the same way as any other change to the tool.
 
-The site is published to the **emscripten-core/emscripten-site** *gh-pages* branch (GitHub pages).
+The site is published to the **kripken/emscripten-site** *gh-pages* branch (GitHub pages).
```
Comment (sbc100) [inline site/source/docs/site/about.rst]: This is the old URL. We don't use `kripken` anymore.  What makes you think we need this change?

Comment (danieloskarsson) [reply site/source/docs/site/about.rst]: https://github.com/kripken/emscripten-site/blob/gh-pages/CNAME

Comment (danieloskarsson) [reply site/source/docs/site/about.rst]: Not sure how github would handle multiple CNAME files referencing the same domain name. I ended up at kripken/emscripten-site as I failed to find a emscripten-core/emscripten-site repository or a gh-pages branch in emscripten-core/emscripten.

**Comment by sbc100** [comment]
Oh I see.  I guess we need to move emscripten-site to the new emscripten org.

**Comment by sbc100** [comment]
I opened a bug to get this moved: https://github.com/emscripten-core/emscripten/issues/8332

**Pull request closed without merging**